### PR TITLE
Fix Issue #84

### PR DESCRIPTION
**Pull request by rderimay**
Moved method DefaultDelegates.getShotOrientation (not used anymore) to
DefaultPointRenderer.getShotOrientation
Added a fallback to this method in the case that
getRawMetadata("orientation") returns nil

### CODE REVIEW

File: focuspoints.lrdevplugin/DefaultPointRenderer.lua [hunk 1]
```diff
@@ -47,8 +46,7 @@ function DefaultPointRenderer.createView(photo, photoDisplayWidth, photoDisplayH
   local userRotation, userMirroring = DefaultPointRenderer.getUserRotationAndMirroring(photo)
 
   -- We read the rotation written in the Exif just for logging has it happens that the lightrrom rotation already includes it which is pretty handy
-  -- We should remove the funcGetShotOrientation later if this is proven to work
-  local exifRotation = DefaultPointRenderer.funcGetShotOrientation(photo, metaData)
+  local exifRotation = DefaultPointRenderer.getShotOrientation(photo, metaData)
```
Comment (musselwhizzle) [inline focuspoints.lrdevplugin/DefaultPointRenderer.lua]: dont you want ```DefaultPointRenderer .getUserRotationAndMirroring``` here?

Comment (musselwhizzle) [inline focuspoints.lrdevplugin/DefaultPointRenderer.lua]: is this variable used anywhere?

**Comment by rderimay** [comment]
No we want this method. 
The exifrotation var is not used anymore, I left it for now just for log purposes in case we would need to understand a bug

**Comment by musselwhizzle** [comment]
ready for it to merge?

**Pull request merged**